jaehoon/src/section04/Book.java

```diff
@@ -0,0 +1,87 @@
+package section04;
+
+import java.util.Objects;
+
+public class Book {
+
+  private String title;
+
+  private String author;
+
+  private String isBn;
```

Comment:
```suggestion
  private String isbn;
```
`ISBN` 처럼 축약어일 경우 굳이 카멜케이스로 변수명을 적을 필요는 없다고 생각하는데, 혹시 이렇게 작성하신 이유가 있을까요?

Reply:
제가 오타였습니다 ㅋㅋ